docs/public/operators-on-minikube.adoc

```diff
@@ -0,0 +1,229 @@
+====
+You can't always install an Operator in the `operators` namespace and expect it to be available across all namespaces. The Operator you're trying to installing needs to be designed to be available in this way as well. Certain Operators only support installation in a single namespace.
+
+Discussing this topic is out of scope of this guide so we have stated it as a note.
+====
+
+Enabling the `olm` addon will, among other things, create an `OperatorGroup` in the `operators` namepsace. Make sure that it's there:
+[source,sh]
+----
+$ kubectl get og -n operators
+NAME               AGE
+global-operators   3m37s
+----
+
+If you don't see one, create it using below command:
+[source,sh]
+----
+$ kubectl create -f - << EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: global-operators 
+  namespace: operators 
+spec:
+  targetNamespaces:
+  - operators
+EOF
+----
+
+Now, install the etcd Operator using below command:
+[source,sh]
+----
+$ kubectl create -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: etcd
+  namespace: operators
+spec:
+  channel: clusterwide-alpha
+  name: etcd
+  source: operatorhubio-catalog
+  sourceNamespace: olm
+  startingCSV: etcdoperator.v0.9.4-clusterwide
+  installPlanApproval: Automatic
+EOF
+----
+
+Give it a few seconds before checking the availability of the etcd Operator:
+[source,sh]
+----
+$ odo catalog list services
+Operators available in the cluster
+NAME                                CRDs
+etcdoperator.v0.9.4-clusterwide     EtcdCluster, EtcdBackup, EtcdRestore
+----
+
+==== Troubleshooting
+
+If you don't see etcd Operator using above command or by doing `kubectl get csv -n operators`, make sure that pod belonging to the `CatalogSource` named `operatorhubio-catalog` is running:
+[source,sh]
+----
+$ kubectl get po -n olm | grep operatorhubio-catalog
+----
+
+If the state of this pod is `CrashLoopBackOff`, delete it so that Kubernetes will automatically spin up a new pod for the `CatalogSource`:
+
+[source,sh]
+----
+$ kubectl delete po -n olm <name-of-operatorhubio-catalog-pod>
+----
+
+Once the pod for this `CatalogSource` is up, wait a few seconds before trying to find the etcd Operator when you do `odo catalog list services`.
+
+=== Installing the Service Binding Operator
+
+We use the link:https://github.com/redhat-developer/service-binding-operator/[Service Binding Operator] to provide `odo link` feature which links an odo component with an Operator backed service. Thus, to be able to use this feature, it's essential that we install the Operator first.
+
+Service Binding Operator is not yet available via the OLM. The team is link:https://github.com/redhat-developer/service-binding-operator/issues/727[working on making it available] through OLM.
+
+At the moment, to make Service Binding Operator work on Kubernetes, we need to keep a couple of things in mind:
+
+. It needs to be installed by cloning its GitHub repository
```

Comment:
I think we do not need that, because we can refer directly all yaml files, demonstrated in line 128.

Reply:
I think this will be taken care of by https://github.com/openshift/odo/pull/4140#issuecomment-729427417